lms/djangoapps/instructor_email_widget/models.py

```diff
@@ -62,6 +62,11 @@ def __unicode__(self):
             self.filter_on,
         )
 
+ORIGIN = (
+    ('E', QueryOrigin.EMAIL),
+    ('G', QueryOrigin.GUI),
+)
+
```

Comment:
Django recommends putting this inside the class: https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.Field.choices